### ISSUE TITLE
Filters on MVP1 - Return Inconsistently

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/Feedback#545]
Sometimes results of Chemicals Categories and ChEBI Classifications, and sometimes they don't.  Is there way to ensure these are more consistent?

Here are some examples in Prod:

ADHD 9/14 12:18PM ET: https://ui.transltr.io/main/results?l=Attention%20Deficit-hyperactivity%20Disorder&i=MONDO:0007743&t=0&q=c6bd8284-84ef-4d59-983f-d624fd6d1da3
![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/cecfe45b-a9b0-44f7-8b49-9de558993b0c)

Ullrich Congenital Muscular Dystrophy 9/13 10:26AM ET: https://ui.transltr.io/main/results?l=Ullrich%20Congenital%20Muscular%20Dystrophy&i=MONDO:0000355&t=0&q=64279692-e688-4649-af40-dc5822013085
![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/df0afe0e-984a-4e36-ad9c-6626395cf948)

Aortic Valve Stenosis 9/12 7:08 AM ET:  https://ui.transltr.io/main/results?l=Aortic%20Valve%20Stenosis&i=MONDO:0042981&t=0&q=1464b264-af50-44f4-90db-f53198abb0b2

Examples in Test:
Common Cold 9/14 12:12 PM ET:  https://ui.test.transltr.io/main/results?l=Common%20Cold&i=MONDO:0005709&t=0&q=e5de966f-9d70-4a5e-b0a4-8ee4c9e841fa
![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/8249b7ac-2673-4e16-8ed9-caff52226a81)

Niemann-pick Disease 9/12 11:08 AM ET:  https://ui.test.transltr.io/main/results?l=Niemann-pick%20Disease&i=MONDO:0001982&t=0&q=b82f8fb3-7125-4eb1-95b3-2a94ad37f4bb
![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/99d82be1-48af-4345-a9fe-d8524983c6de)

Liver Disorder 9/11 2:45 PM ET https://ui.test.transltr.io/main/results?l=Liver%20Disorder&i=MONDO:0005154&t=0&q=0f76404f-e236-4a46-8bbe-d8c5f867cefb

Examples in CI:

ADHD Inattentive Type 9/14 12:19PM ET : https://ui.ci.transltr.io/main/results?l=Attention%20Deficit%20Hyperactivity%20Disorder,%20Inattentive%20Type&i=MONDO:0005302&t=0&q=b39f7232-0bec-42e6-8e47-9a18289f7d96
![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/8c36a8b4-256a-4d36-b20f-055a8480b567)

Aortic Valve Stenosis 9/12 6:53 AM ET  https://ui.ci.transltr.io/main/results?l=Aortic%20Valve%20Stenosis&i=MONDO:0042981&t=0&q=2a167dcf-4994-49a2-8a08-b57b8b667e8b
![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/40b17151-f6bd-4498-8a95-c53c1d734f36)

**Comment by gprice1129** [comment]
This is an issue with the annotation service. If it takes too long or is not working, then we do not have enough information to construct the facets. In order to solve this problem the annotation service must be very stable and be fast under load. @newgene

**Comment by newgene** [comment]
@gprice1129 this is consistent with what we observe, we are working with the ITRB team to resolve the issue (most likely needs a scaling up).

And @gprice1129 we know ARS calls annotator service directly, the access you mentioned above is referring to that ARS-level access or the direct access from the UI application? If UI's direct access, can you let me know how the service was called, GET for a single curie ID at a time or a POST for a batch of curie IDs?

**Comment by gprice1129** [comment]
@newgene it is direct access from the UI application. We call it with a POST with a batch of empty curie IDs.

**Comment by newgene** [comment]
@gprice1129 got it. I assume you did not mean `empty` curie IDs?

**Comment by gprice1129** [comment]
I just mean nodes with no attributes

**Comment by newgene** [comment]
Yes, that will reduce the bandwidth needed over the network, understand now. Thanks!

As an update, ITRB team is working on scaling-up the current Service Provider deployment (where annotator service is hosted).

**Comment by newgene** [comment]
@gprice1129 @Genomewide ITRB has made a new deployment of annotator prod, let's see the new scaled-up deployment has enough capacity to handle the incoming requests. And of course, the optimization of how ARS and UI call annotator would help significantly regardless.

**Comment by gprice1129** [comment]
@sstemann can you retest and see if it is more consistent?

**Comment by newgene** [comment]
@sstemann I know ITRB team is fixing a deployment issue this morning, once it's done, I will let you know here to give another try.

Annotator had some down time early this morning, it's already back, but some deployment changes are expected to be made on prod today, may impact the service briefly.

**Comment by newgene** [comment]
@sstemann fyi, ITRB has done the deployment change, feel free to give another try.
